src/module/apps/tracker.js

```diff
@@ -200,28 +200,83 @@ export class STATracker extends Application {
    * @param {STATracker.Resource} resource The resource to modify.
    * @param {number} value The value to set for the given resource.
    */
-  static async DoUpdateResource(resource, value) {
+
+// Store the accumulated changes and a timer
+  static accumulatedChanges = {
+    momentum: 0,
+    threat: 0
+  };
+
+  static chatMessageTimeout = null; // Holds the timeout for debouncing

```

Comment:
This introduces an ESLint error, because it's a feature introduced after ES2018 (which is what our Linter is configured for).

I'm recommending ignoring this issue; static class members are very much widely supported in Desktop environments.  I'm working on a PR to update our ESLint config to a more up-to-date standard.